etc/sync-function-template.js

```diff
@@ -605,6 +605,65 @@ function synctos(doc, oldDoc) {
     }
   }
 
+  function resolveCollectionItems(originalItems, itemPrefix) {
```

Comment:
I went with the more general-purpose "resolve" because, in the case where there is only a single item rather than a list/array, it is will transparently morph that single item into a list/array.